src/cpu/pred/BranchPredictor.py

```
@@ -1060,16 +1060,33 @@ class BTBTAGE(TimedBaseBTBPredictor):
-    TTagBitSizes = [16]
-    TTagPcShifts = [1]
-
-    histLengths = [16]
-    numDelay = 0
+class MicroTAGE(TimedBaseBTBPredictor):
+    """Micro-sized BTB TAGE predictor used alongside uBTB"""
+    type = 'MicroTAGE'
+    cxx_class = 'gem5::branch_prediction::btb_pred::MicroTAGE'
+    cxx_header = "cpu/pred/btb/microtage.hh"
+
+    needMoreHistories = Param.Bool(True, "MicroTAGE needs more histories")
```

Ruff (0.14.14)

`Param` may be undefined, or defined from star imports

(F405)